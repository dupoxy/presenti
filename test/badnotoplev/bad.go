/* #nb:
A Bad presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/
*/

// A NOT top level comment Bad Example
package main

import (
	"fmt"
)

/* #nb: * Bla bla */

/* #nb: * Bla bla
 */

func main() {
	
	/* #nb: THIS IS NOT a top level comment
	Bla bla */
	
	fmt.Println("bad")
}
