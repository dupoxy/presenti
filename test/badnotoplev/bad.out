# DO NOT EDIT GENERATED by presenti
A Bad presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/

* Bla bla
* Bla bla

